src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1169.mdx

```diff
@@ -8,7 +8,7 @@ version: '1169'
 
 ### Privacy: Added the ability for admin users to toggle cookie collection On/Off via Browser Application Settings UI
 
-The agent now accepts new configuration `privacy.cookies_enabled`. When it is set to false (OFF), the agent doesn't write any cookies ([JSESSIONID](https://docs.newrelic.com/docs/browser/new-relic-browser/page-load-timing-resources/new-relic-cookies-used-browser#jsessionid) & [NREUM](https://docs.newrelic.com/docs/browser/new-relic-browser/page-load-timing-resources/new-relic-cookies-used-browser#nreum)) and it also notifies the intake server to not return a cookie. This enables customers to comply with [GDPR/PECR ICO guidelines](https://ico.org.uk/for-organisations/guide-to-pecr/cookies-and-similar-technologies/) around cookies. By default, all current and newly created applications will have this value set to true (ON), so there will be no change in the data currently being collected by the agent. Admin users must toggle the setting to OFF and redeploy the agent in order to see this change take effect. Once cookie collection is set to false, users should no longer see session information captured with their event data. If a session was already in progress, that data will continue to be collected until that session expires. New sessions should report a null value.
+The agent now accepts new configuration `privacy.cookies_enabled`. When it is set to false (OFF), the agent doesn't write any third party cookies ([JSESSIONID](https://docs.newrelic.com/docs/browser/new-relic-browser/page-load-timing-resources/new-relic-cookies-used-browser#jsessionid) & [NREUM](https://docs.newrelic.com/docs/browser/new-relic-browser/page-load-timing-resources/new-relic-cookies-used-browser#nreum)) and it also notifies the intake server to not return a third party cookie. By default, all current and newly created applications will have this value set to true (ON), so there will be no change in the data currently being collected by the agent. Admin users must toggle the setting to OFF and redeploy the agent in order to see this change take effect. Once cookie collection is set to false, users should no longer see session information captured with their event data. If a session was already in progress, that data will continue to be collected until that session expires. New sessions should report a null value.
```

Comment:
The agent now accepts new configuration 'privacy....'

This reads almost like a fragment, but I'm not sure the best way to fix it. "the configuration", "the 'privacy...' configuration", "configuration of 'privacy...'", something like that

Reply:
It is > it's

Reply:
to not return > not to return

Reply:
there will be no change in the data currently being collected... > there won't be a any changes to the data the agent collects

Reply:
Is it redeploy or re-deploy?

Reply:
that data will continue to be collected until > the agent will continue to collect data until